### DISCUSSION TITLE
Tag templates

### DISCUSSION THREAD

**Discussion by filcuk** [almarklein/timetagger#367] [Ideas]
Hi, I've been making great use of this tool lately.

I would really like to suggest creating tag 'templates', where, for example, project tags would always use the same colour, e.g. setting `#222222` for `#project/` would apply to `#project/xyz` or `#project/123` automatically.

**Comment by almarklein** [comment]
Hi!

Thanks for your suggestion! If I understand it correctly, you'd like to be able to set the color of tag `#project`, which would then result in all `#project/xx` to have the same color?

**Reply by filcuk** [reply]
Hi, that's exactly right!
Possibly also inheriting whether it's a minor or major tag.

**Comment by almarklein** [comment]
I turned this into an issue: https://github.com/almarklein/timetagger/issues/368
